src/main/java/codeu/model/data/User.java

```diff
@@ -36,11 +65,7 @@
    * @param description description to show on user's profile page
    */
   public User(UUID id, String name, String password, Instant creation, String description) {
-      this.id = id;
-      this.name = name;
-      this.password = password;
-      this.creation = creation;
-      this.description = description;
+      this(id, name, password, creation, description, false);
```

Comment:
Why is this constructor stylized differently than the one above? Should each parameter also have a new line?

Reply:
I didn't put each argument in new line because they still fit in the 100 characters limit per line. For the same reason, I put each parameter in new line for the first constructor.